client/src/components/menubar/index.tsx

```diff
@@ -277,171 +291,154 @@ class MenuBar extends React.PureComponent<MenuBarProps, State> {
     ];
 
     return (
-      <div
-        style={{
-          display: "flex",
-          flexDirection: "row",
-          flexWrap: "wrap",
-          justifyContent: "space-between",
-          width: "100%",
-        }}
-        data-test-id="menubar"
-      >
-        <div
-          style={{
-            display: "flex",
-            flexDirection: "row",
-            flexWrap: "wrap",
-            justifyContent: "left",
-            marginTop: 8,
-          }}
-        >
+      <MenuBarWrapper data-test-id="menubar">
+        <EmbeddingWrapper>
```

Comment:
So much cleaner and readable 😭 ✨ Thanks so much for doing this, @rainandbare 🤩 👏